recipes/mpi-serial/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "mpi-serial" %}
```

Comment:
I get the impression that we might make our lives a lot easier if we name the package `mpi_serial` instead of `mpi-serial`.  This is because there seems to be a lot of MPI-related infrastructure that doesn't distinguish between the package name and the its name in the build string.  Since the build string has to be `mpi_serial`, the package name probably does, too.

Reply:
See https://github.com/conda-forge/mpi-feedstock/pull/10/files

Reply:
Another option would be `mpiserial` (no punctuation).

Reply:
Agreed. Since the mpi package calls it `mpi_serial`, let's stick with that.